tests/migrations/test_writer.py

```diff
@@ -647,3 +647,11 @@ def deconstruct(self):
 
         string = MigrationWriter.serialize(models.CharField(default=DeconstructibleInstances))[0]
         self.assertEqual(string, "models.CharField(default=migrations.test_writer.DeconstructibleInstances)")
+
+    def test_force_construct_class_arguments(self):
+        """
+        Unserializable values can be migrated anymore
+        """
+
+        string = MigrationWriter.serialize(models.CharField(default=Unserializable))[0]
+        self.assertEqual(string, "models.CharField(default=migrations.models.Unserializable)")
```

Comment:
The model was declared as `default=Unserializable()` but it's serialized without the parentheses?